pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/README.md

```diff
@@ -0,0 +1,94 @@
+# Python vs. PostgresML
+
+## PostgresML
+
+```
+cargo pgx run --release
+```
+
+### Schema
+
+```postgresql
+CREATE TABLE public.flights_delay_mat (
+    depdelayminutes real,
+    year real NOT NULL,
+    quarter real NOT NULL,
+    month real NOT NULL,
+    distance real NOT NULL,
+    dayofweek real NOT NULL,
+    dayofmonth real NOT NULL,
+    flight_number_operating_airline real NOT NULL,
+    originairportid real NOT NULL,
+    destairportid real NOT NULL,
+    tail_number real NOT NULL
+);
+```
+
+### Data
+
+```bash
+curl -L -o ~/Desktop/flights.csv https://hyperparam-assets-public.s3.us-west-2.amazonaws.com/benchmarks/flights.csv
+```
+
+```postgresql
+\copy flights_delay_mat FROM '~/Desktop/flights.csv' CSV HEADER;
+```
+
+### Train
+
+```postgresql
+SELECT * FROM pgml.train(
+	'r2',
+	'regression',
+	'flights_delay_mat',
+	'depdelayminutes',
+	algorithm => 'xgboost',
+	hyperparams => '{ "n_estimators": 25 }'
```

Comment:
It'd be interesting to see how scaling the estimators impacts performance in both (10/100/1000 estimators)

Reply:
More estimators, more problems.